docs/data/base/components/select/select.md

```diff
@@ -90,17 +90,14 @@ For the sake of brevity, the rest of the demos throughout this doc will not use
 
 ### Multi-select
 
-The Unstyled Multi-Select component lets your users select multiple options from the list.
-
-```js
-import { MultiSelectUnstyled } from '@mui/base/SelectUnstyled';
-```
+The Unstyled Select component lets your users select multiple options from the list.
```

Comment:
I don't know what we ended up in the styled guide, but "Unstyled Select" looks awkward. It should either be "`SelectUnstyled`" or "unstyled select", or even "unstyled Select". I don't think in either case we should uppercase "unstyled" as a word.

Reply:
From what I remember we agreed on "Unstyled Select" as it's a proper noun. cc @samuelsycamore

Reply:
Alright, thanks for linking the style guide 👍